vue/src/router/index.js

```diff
@@ -9,6 +10,10 @@ const routes = [
     path: "/",
     component: Index,
   },
+  {
+    path: "/game-playing/:id",
+    component: GamePlaying,
+  },
```

Comment:
This route should use dynamic import to enable lazy-loading (More detail: https://router.vuejs.org/guide/advanced/lazy-loading.html).

However, I'm not sure if it works on starport (It should work right?)

Reply:
Yes, it worked